CMGTools/TTHAnalysis/python/analyzers/ttHAlphaTVarAnalyzer.py

```diff
@@ -1,3 +1,6 @@
+from PhysicsTools.Heppy.analyzers.core.Analyzer import Analyzer
+from PhysicsTools.HeppyCore.utils.deltar import deltaR, deltaPhi
```

Comment:
@emanueledimarco These lines are the same as 11 and 12? This addition isn't necessary I don't think